ephios/core/templatetags/user_extras.py

```diff
@@ -74,3 +74,8 @@ def shifts_needing_disposition(user):
         )
         .filter(request_count__gt=0)
     )
+
+
+@register.filter(name="allows_grant_for")
+def allows_grant_for(grant_group_ids, user_group_ids):
+    return bool(grant_group_ids & user_group_ids)
```

Comment:
Now I see why you picked the name but just looking at this line something like `intersects` might be the better name for the filter